server/asset/btc/btc.go

```diff
@@ -363,6 +379,51 @@ func (btc *Backend) blockInfo(verboseTx *btcjson.TxRawResult) (blockHeight uint3
 	return
 }
 
+// anylist is a list of RPC parameters to be converted to []json.RawMessage and
+// sent via RawRequest.
+type anylist []interface{}
+
+// call is used internally to  marshal parmeters and send requests to  the RPC
+// server via (*rpcclient.Client).RawRequest. If `thing` is non-nil, the result
+// will be marshaled into `thing`.
+func (btc *Backend) call(method string, args anylist, thing interface{}) error {
+	params := make([]json.RawMessage, 0, len(args))
+	for i := range args {
+		p, err := json.Marshal(args[i])
+		if err != nil {
+			return err
+		}
+		params = append(params, p)
+	}
+	b, err := btc.node.RawRequest(method, params)
+	if err != nil {
+		return fmt.Errorf("rawrequest error: %v", err)
+	}
+	if thing != nil {
+		return json.Unmarshal(b, thing)
+	}
+	return nil
+}
+
+// getBlockchainInfoResult models the data returned from the getblockchaininfo
+// command.
+type getBlockchainInfoResult struct {
+	Blocks               int64  `json:"blocks"`
+	Headers              int64  `json:"headers"`
+	BestBlockHash        string `json:"bestblockhash"`
+	InitialBlockDownload bool   `json:"initialblockdownload"`
+}
+
+// getBlockchainInfo sends the getblockchaininfo request and returns the result.
+func (btc *Backend) getBlockchainInfo() (*getBlockchainInfoResult, error) {
+	var chainInfo *getBlockchainInfoResult
+	err := btc.call(methodGetBlockchainInfo, nil, &chainInfo)
```

Comment:
Here and in client asset, you're using two levels of indirection (`**getBlockchainInfoResult`) for the final input arg to `call`.  That might work but we're not doing that in the other calls.